.github/workflows/kuksa_databroker_build.yml

```diff
@@ -126,26 +115,28 @@ jobs:
       working-directory: ${{github.workspace}}/
       run: |
         which cross || cargo install cross
+        cargo install cargo-license cargo-cyclonedx
+        pip install "git+https://github.com/eclipse-kuksa/kuksa-common.git@6f3d7627760582d8ba83cc8a0f7449d00fffee84#subdirectory=sbom-tools"
     - name: Build
       working-directory: ${{github.workspace}}/
+      env:
+        KUKSA_DATABROKER_FEATURES: databroker/viss,databroker/tls
```

Comment:
How does features works for the CLI? Right now we do not set the same features for CLI - intentional? or is CLI independent from the features, i.e. support VISS/Websocket and TLS anyway?